### DISCUSSION TITLE
Connect to Lubuntu without Display

### DISCUSSION THREAD

**Discussion by berndver** [rustdesk/rustdesk#10289] [Q&A]
Does anyone manage to connect to a Lubuntu 22 client without a display connected? 
I tried the instructions for Headless Linux, but this does not work for me. I dont have the gnome Desktop environment. If i create a xorg.conf my System wont Boot.
Maybe someone has a solution without buying a dummy plug.
Thanks for Your help!

**Comment by jacksonmacd** [comment]
No answer for you, but I think I have the same issue. Remote computer is a Raspberry Pi 4 running Ubuntu Desktop. I have set the "Use linux headless" to True in the GUI, and I disabled Wayland through a configuration file. The "controlling" computer is Windows 11. Works fine when the remote Linux computer has a connected monitor, but always stops when the monitor is disconnected. If I subsequently connect the monitor, RD is stalled at a dialog that wants to select the active monitor.
